libs/table.lua

```diff
@@ -313,29 +313,39 @@ function ext_table.shuffle(tbl)
 	return tbl
 end
 
----Returns a random key, value index from an array.
+---Returns a random (key, value) pair selected from the sequence portion of a table, or `nil` on an empty array.
 ---
 ---This function only works on the sequence portion of the table. Its behavior is undefined if the table has holes.
 ---@param tbl table
----@return any, any
+---@return any|nil, any|nil
```

Comment:
`any|nil` is just `any`

Reply:
Resolved. Although, I have actually intentionally left that there knowing it is already implied in `any`.  It is really useful to leave the additional `|nil` or `?` when generating docs from those comments, as you can easily look for an explicit `|nil` for tagging it as optional/nilable.